docs/configure/styling-and-css.md

```diff
@@ -22,66 +32,136 @@ Storybook is pre-configured to recognize imports for CSS files. To add global CS
+Storybook recreates your Next.js configuration, so you can use CSS modules in your stories without any extra configuration.
 
-## CSS-in-JS
+</Callout>
+</IfRenderer>
+
+## PostCSS
+
+### Vite
+Vite comes with PostCSS support out-of-the-box. If you have customized the PostCSS configuration in your `vite.config.js` this will automatically be applied to your Storybook as well. Read more about [Vite's PostCSS support](https://vitejs.dev/guide/features.html#postcss).
+
+### Webpack
+<IfRenderer renderer='react'>
+<Callout
+  variant="info"
+  icon="📣"
+  title="Using `@storybook/nextjs`?"
+>
+
+Storybook recreates your Next.js configuration, so you can use PostCSS in your stories without any extra configuration.
+</Callout>
+</IfRenderer>
+
+## CSS pre-processors
 
+### Vite
+Vite comes with Sass, Less, and Stylus support out-of-the-box. Read more about [Vite's CSS Pre-processor support](https://vitejs.dev/guide/features.html#css-pre-processors).
+
+### Webpack
+<IfRenderer renderer='react'>
+<Callout
+  variant="info"
+  icon="📣"
+  title="Using `@storybook/nextjs`?"
+>
+
+Storybook recreates your Next.js configuration, so you can use Sass in your stories without any extra configuration.
+
+</Callout>
+</IfRenderer>
+
+However, if you're using Webpack and want to use Sass or less, you'll need some extra configuration. We recommend installing [`@storybook/addon-styling-webpack`](https://storybook.js.org/addons/@storybook/addon-styling-webpack/) to help you configure these tools. Or if you'd prefer, you can customize [Storybook's webpack configuration yourself](../builders/webpack.md#override-the-default-configuration) to include the appropriate loader(s).
+
+## CSS-in-JS
 CSS-in-JS libraries are designed to use basic JavaScript, and they often work in Storybook without any extra configuration. Some libraries expect components to render in a specific rendering “context” (for example, to provide themes), which can be accomplished with `@storybook/addon-themes`'s [`withThemeFromJSXProvider` decorator](https://github.com/storybookjs/storybook/blob/next/code/addons/themes/docs/api.md#withthemefromjsxprovider).
 
 ## Adding webfonts
 
+### `.storybook/preview-head.html`
 If you need webfonts to be available, you may need to add some code to the [`.storybook/preview-head.html`](./story-rendering.md#adding-to-head) file. We recommend including any assets with your Storybook if possible, in which case you likely want to configure the [static file location](./images-and-assets.md#serving-static-files-via-storybook-configuration).
 
-<IfRenderer renderer='angular'>
+### `.storybook/preview.ts`
+If you're using something like [`fontsource`](https://fontsource.org/) for your fonts, you can import the needed css files in your [`.storybook/preview.ts`](./overview.md#configure-story-rendering) file.
 
-## Troubleshooting
+</IfRenderer>
```

Comment:
I couldn't find the opening tag for this?

Reply:
That's for the top level "not angular" tag